types/config.go

```diff
@@ -22,6 +22,7 @@ type Config struct {
 	DebugUser       string        `split_words:"true" required:"false"` // when set, only accept commands from this user
 	NoInitSync      bool          `split_words:"true" required:"false"` // if set, does not run database role sync on init
 	NoDatabase      bool          `split_words:"true" required:"false"` // if set, does not connect to database
+	WikiURL         string        `split_words:"true" required:"false"` // wiki git url
```

Comment:
You can add a default here using [envconfig](https://github.com/kelseyhightower/envconfig) `default` tag

```go
	WikiURL         string        `split_words:"true" required:"false" default:"https://github.com/openmultiplayer/wiki"` // wiki git url
```

Reply:
This will save me having to reconfigure the service before deployment!